pkgs/development/tools/language-servers/nixd/default.nix

```diff
@@ -32,6 +32,7 @@ let
       meson
       ninja
       pkg-config
+      llvmPackages.llvm
```

Comment:
I don't know why llvm is required for all packages. Maybe just adding this item for libnixt would be sufficient?

Reply:
Actually, it isn't needed for `libnixt`, it is just needed for `nixd`. You OK with the current version, where I just override the native build inputs for `nixd`?